tests/unit/test_config.py

```diff
@@ -110,3 +110,19 @@ def test_base_image(image: str, expected_base_image: BaseImage):
     assert: expected base image is returned.
     """
     assert BaseImage.from_str(image) == expected_base_image
+
+
+@pytest.mark.parametrize(
+    "base_image, expected_version",
+    [
+        pytest.param(BaseImage.JAMMY, "22.04", id="jammy"),
+        pytest.param(BaseImage.NOBLE, "24.04", id="22.04"),
```

Comment:
The id does not look right here.

Reply:
You must be a good fisher with all these great catches!